### PR TITLE
Create tables based on diagram and create join table for m:m relation.

### PR DESCRIPTION
**Pull request by brahimdidi**
In this part, I created a database schema based on a given diagram.

## Requierments
Create a file named schema_based_on_diagram.sql where you implement the database from the diagram.
Join tables from many-to-many relationships might not appear in the diagram, but you still need them.
Remember to add the FK indexes.

### CODE REVIEW

File: shema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  status VARCHAR(100)
+);
+
+CREATE TABLE invoices (
+  id BIGSERIAL NOT NULL  PRIMARY KEY,
+  total_amount DECIMAL,
+  admitted_at TIMESTAMP,
+  payed_at TIMESTAMP,
+  medical_histories_id BIGINT NOT NULL UNIQUE REFERENCES medical_histories
+);
```
Comment (thecodechaser) [inline shema_based_on_diagram.sql]: - [ ] Please change the `admitted_at` column with `generated_at` to follow the diagram schema.

image

File: shema_based_on_diagram.sql [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  name VARCHAR(100),
+  date_of_birth DATE
+);
+CREATE TABLE medical_histories (
+  id BIGSERIAL NOT NULL PRIMARY KEY,
+  admitted_at TIMESTAMP,
+  patient_id BIGINT NOT NULL REFERENCES patients(id),
+  status VARCHAR(100)
+);
+
```
Comment (thecodechaser) [inline shema_based_on_diagram.sql:12]: - [ ] Please add FOREIGN KEYS indexes as requested in the project requirements. hint:
image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @brahimdidi, @SiansandaTamara👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by danmainah — APPROVED** [review]
Hi @brahimdidi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights

✔️ No linter errors
✔️ Gitflow was correctly used

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**